client/my-sites/subscribers/mutations/use-subscriber-remove-mutation.ts

```diff
@@ -36,61 +36,81 @@ const useSubscriberRemoveMutation = (
 	} );
 
 	return useMutation( {
-		mutationFn: async ( subscriber: Subscriber ) => {
-			if ( ! siteId || ! subscriber ) {
+		mutationFn: async ( subscribers: Subscriber[] ) => {
+			if ( ! siteId || ! subscribers || ! subscribers.length ) {
```

Comment:
Do we need a maximum length here? Or the most per page is 100 subscribers, so that's the highest number that could be deleted at once?